modules/ossm-control-plane-configuration-migration-requirements.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-cluster-wide-migration_{context}"]
+= Control plane configuration migration requirements
+
+During the migration process, two cluster-wide control planes run in the same cluster while the data plane namespaces are gradually migrated to the {SMProductName} 3.0 installation. One control plane is associated with the {SMProductName} {SMv2Version} installation and the other is associated with the {SMProduct} 3.0 installation. You must carefully plan out the migration steps to avoid possible conflicts between the two control planes.
+
+[id="ossm-cluster-wide-migration-root-certificate_{context}"]
+== Root certificate
+
+During the migration, both control planes must share a root certificate. This can be achieved by installing the 3.0 control plane into the same namespace as 2.6 control plane. The migration procedures show how to verify that the root certificate is shared.
```

Comment:
```suggestion
During the migration, both control planes must share a root certificate. To share a root certificate between both control planes, you install the 3.0 control plane into the same namespace as 2.6 control plane. The migration procedures show how to verify that the root certificate is shared.
```
- Revised to avoid a potentially ambiguous "This", per IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references

Reply:
done